flink-ai-flow/ai_flow_plugins/job_plugins/flink/flink_env.py

```diff
@@ -29,7 +35,7 @@ class FlinkEnv(object):
     """
 
     @abstractmethod
-    def create_env(self) -> (ExecutionEnvironment, TableEnvironment, StatementSet):
+    def create_env(self) -> (ExecutionEnvironment, WrappedTableEnvironment, WrappedStatementSet):
```

Comment:
From an API perspective, the `create_env` interface should not contain the `WrappedTableEnvironment`,  `WrappedStatementSet`, which could move to the `flink_wrapped_env.py`.

Reply:
fix with https://github.com/alibaba/flink-ai-extended/pull/477/commits/2eb4dea40f4d9841bb7cc8117be46ba5c08911e6